### DISCUSSION TITLE
How to express multiple conditional statements？

### DISCUSSION THREAD

**Discussion by liujin1321** [yalmip/YALMIP#1275] [Optimization modelling]
I would like to ask if we can implement the nesting of two conditional statements using implicit syntax. For example:
![image](https://github.com/yalmip/YALMIP/assets/117717679/fe9ce46f-fe04-4110-800c-9ae4822e1d53)
I can successfully create a condition statement, but for the nested condition statement, I do not know how to do it using implies.
Looking forward to your reply！

**Comment by johanlofberg** [comment]
As always, just untangle it

you have 4 cases which you introduce binaries for and then simply impose those conditions

case1 A>=0, B>=0, whatever
case2 A>=0, B<=0, whatever
case 3 A<=0, D>=0, whatever
case 4 A<=0, D<=0, whatever

**Reply by liujin1321** [reply]
Thank you very much for your answer and guidance.
I would also like to ask whether it is possible to achieve multi-objective optimization. If I want to minimize costs and carbon emissions, how do I do it?

**Reply by johanlofberg** [reply]
https://yalmip.github.io/multi-objective-problems
